prow/config.yaml

```diff
@@ -63,6 +63,41 @@ tide:
 
 presubmits:
   tektoncd/plumbing:
+  - name: pull-dogfooding-mario-test
+    agent: kubernetes
+    always_run: true
```

Comment:
I think we should leave this as an "on-demand" only job - i.e. do not run unless someone does "/test pull-dogfooding-mario-test"

Reply:
make sense